prometheus/output_formatter_test.cc

```diff
@@ -8,16 +8,18 @@ namespace {
   using namespace prometheus;
   using namespace prometheus::impl;
 
+  MetricFamilyPtr make_family() { return MetricFamilyPtr( new MetricFamily()); }
```

Comment:
Since we're going to keep "metricfamily" in the API for now, I think this should be make_metricfamily.